setup_env.sh

```diff
@@ -0,0 +1,7 @@
+export ARCH_NAME=grayskull                                                                   
+export TT_METAL_HOME=~/Desktop/devs/tt-metal/
```

Comment:
I'm not super sure this should be in the repository. Foremost, the path is for your specific setup and not everyone have tt-metal on their dekstop.

I'm more leaning towards removing this file for the sake of generality. What's your thought?